src/Inventory/Home.js

```diff
@@ -35,14 +37,49 @@ export default class Home extends Component {
 						color={colors.tint}
 						size={24}
 						underlayColor='transparent'
-						onPress={() => navigation.navigate('CreateList')}
+						onPress={params.showNewListInput}
 						containerStyle={styles.rightButton}
 					/>
 				</View>
 			)
 		}
 	}
 
+	componentWillMount() {
+		this.props.navigation.setParams({ showNewListInput: this.showNewListInput });
+	}
+
+	showNewListInput() {
+		this.setState(prevState => ({ newListInputVisible: !prevState.newListInputVisible }));
```

Comment:
Can we put this on multiple lines for readability?

(And perhaps I do this too... if so I'll update the stuff I wrote 🙂)